nx/include/switch/services/fs.h

```diff
@@ -77,13 +77,26 @@ typedef struct
     u64 unk_x38;                    ///< 0 for SystemSaveData/SaveData.
 } FsSave;
 
+/// SaveDataExtraData Struct
+typedef struct {
+    FsSave save;      ///< Save struct.
+    u64 owner_id;     ///< Title id of the owner of this save data. 0 for SystemSaveData.
```

Comment:
ownerId

Reply:
Actually: https://github.com/switchbrew/libnx/blob/master/CODESTYLE.md#struct-members

Reply:
Well, the other thing is that most of the structs in fs.h don’t use that style (and it’s a similar case in other headers too). Kinda hard to tell which one to use tbh.